index.js

```diff
@@ -122,7 +122,22 @@ function parse (opts, cb) {
       file = resolve.sync(filename, { basedir: path.dirname(file) })
     }
 
-    fs.readFile(file, 'utf8', read)
+    if (opts.transformer) {
+    	var cmd = opts.transformer.split(" ") ; 
+    	var args = cmd.slice(1).map(function(a){ return a=="$$"?file:a }) ;
+    	var p = require('child_process').spawn(cmd[0],args) ;
+    	var output = "" ;
+    	p.stdout.on('data',function(data){
+    		output += data.toString() ;
```

Comment:
using string concatenation like this could result in multibyte unicode characters getting converted incorrectly (as they could span across multiple buffers). you should use http://npmjs.org/concat-stream here instead